website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -86,6 +86,8 @@ The following are the optional configurations you can set in <Constant name="clo
 | [Google cloud storage-bucket](#google-cloud-storage-bucket)    | Location for storing objects in Google Cloud Storage   | String  | `my-bucket`                 |
 | [Dataproc region](#dataproc-region)                | Specifies the cloud region for running data processing jobs    | String  | `US`, `EU`, `asia-northeast1` |
 | [Dataproc cluster name](#dataproc-cluster-name)          | Assigns a unique identifier to a group of virtual machines in Dataproc          | String  | `my-cluster`                |
+| [Notebook Template ID](#dataproc-cluster-name)          | Unique identifier to a Colab Enterprise notebook runtime          | Integer  | `7018811640745295872`                |
```

Comment:
The links on these two look like they might have fallen victim to copy/paste errors.

Reply:
What link?

Reply:
[Notebook Template ID](#dataproc-cluster-name) will say "Notebook Template ID" but will take you to the "dataproc-cluster-name" section.

Reply:
```suggestion
| [Notebook Template ID](#notebook-template-id)          | Unique identifier to a Colab Enterprise notebook runtime          | Integer  | `7018811640745295872`                |
```

Reply:
oh of course - good catch!